web-app/js/portal/filter/ui/NumberFilterPanel.js

```diff
@@ -101,6 +101,12 @@ Portal.filter.ui.NumberFilterPanel = Ext.extend(Portal.filter.ui.BaseFilterPanel
 
         if (this.firstField.validate() && this.secondField.validate()) {
 
+            // Fixes #1696
+            if (this._operatorIsBetween() && 
```

Comment:
I also notice that we have logic like this further down (https://github.com/aodn/aodn-portal/pull/1729/files#diff-0e56b1024e0670eba5465784a3b77ebdR150). I don't think there's any need to have it twice, and where you have the logic (within `_updateFilter`) seems to be the better place for it. So we can remove 4 variables and an if-condition from the further down block and just rely on the logic you've got in _updateFilter.